packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -80,13 +77,16 @@ const defaultConfig: Partial< SnapWPEnv & SnapWPConfig > = {
  */
 const envConfig = (): Partial< SnapWPEnv > => ( {
 	/* eslint-disable n/no-process-env -- These are the env variables we want to manage. */
-	nextUrl: process.env.NEXT_PUBLIC_URL,
-	homeUrl: process.env.NEXT_PUBLIC_WORDPRESS_URL,
 	graphqlEndpoint: process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT,
-	uploadsDirectory: process.env.NEXT_PUBLIC_WORDPRESS_UPLOADS_PATH,
-	restUrlPrefix: process.env.NEXT_PUBLIC_WORDPRESS_REST_URL_PREFIX,
-	useCorsProxy: process.env.NEXT_PUBLIC_USE_CORS_PROXY === 'true',
-	corsProxyPrefix: process.env.NEXT_PUBLIC_CORS_PROXY_PREFIX,
+	hasCorsProxy: process.env.NEXT_PUBLIC_USE_CORS_PROXY,
```

Comment:
https://github.com/rtCamp/snapwp/pull/111#discussion_r1987710723

```suggestion
	hasCorsProxy: !! process.env.NEXT_PUBLIC_CORS_PROXY_PREFIX,
```